test/e2e/tests/test_ca_activation.py

```diff
@@ -38,21 +38,21 @@
 def create_secret(k8s_secret):
     secret = k8s_secret(
         "default",
-        "certificate-secret",
+        "certificate-secret-2",
         "certificate",
         "value"
     )
     yield secret
 
 @pytest.fixture(scope="module")
-def test_create_ca(acmpca_client):
+def create_ca():
```

Comment:
Thoughts: You might want to reuse the function declared in `test_ca.py` - however to inject new ORG/COMMON_NAME parameters you might want to use something like pytest request markers https://docs.pytest.org/en/7.1.x/example/markers.html